pyscriptjs/src/main.ts

```diff
@@ -210,22 +210,21 @@ export class PyScriptApp {
         //Refresh the module cache so Python consistently finds pyscript module
         runtime.invalidate_module_path_cache()
 
-        // inject `define_custom_element` it into the PyScript module scope
+        // inject `define_custom_element` and showWarning it into the PyScript
+        // module scope
         const pyscript_module = runtime.interpreter.pyimport('pyscript');
         pyscript_module.define_custom_element = define_custom_element;
-        pyscript_module.PyScript.set_version_info(version);
+        pyscript_module.showWarning = showWarning;
+        pyscript_module._set_version_info(version);
         pyscript_module.destroy();
 
-        // TODO: Currently adding the imports for backwards compatibility, we should
-        //       remove it
+        // import some carefully selected names into the global namespace
         await runtime.run(`
-        from pyscript import *
-        `);
-        logger.warn(`DEPRECATION WARNING: 'micropip', 'Element', 'console', 'document' and several other \
-objects form the pyscript module (with the exception of 'display') will be \
-be removed from the Python global namespace in the following release. \
-To avoid errors in future releases use import from pyscript instead. For instance: \
-from pyscript import micropip, Element, console, document`);
+        import js
```

Comment:
Do we need/want to import the `js` module here? I know it's one of the modules imported 'by default' previously, but I wonder if want to encourage users to import it themselves (i.e. make it a DeprecatedGlobal). Open to opinions though.

Reply:
This is a valid point. We don't even need to make it a `DeprecatedGlobal` because it was not available before.
I'm +0.5 to remove it and require to import it explicitly.
The same for `pyscript`, I'd very happy to require `import pyscript`.
Let's see what @fpliger thinks

Reply:
commenting for posterity since we merged this already...

I'm fine to leave this as is, sit down and think this a bit more after the release...